docs/source/migrationGuide2.0.rst

```diff
@@ -38,48 +73,95 @@ The procedural interfaces for BinaryErode, BinaryDilate and BinaryClosingByRecon
 |  seg = sitk.BinaryDilate(seg, 5)                         |  seg = sitk.BinaryDilate(seg, [5]*seg.GetDimension()) |
 +----------------------------------------------------------+-------------------------------------------------------+
 
-Transformations
-+++++++++++++++
+The follow is a list of procedures affected by this change:
+
+* `BinaryClosingByReconstruction`
+* `BinaryDilate`
+* `BinaryErode`
+* `BinaryMorphologicalClosing`
+* `BinaryMorphologicalOpening`
+* `BinaryOpeningByReconstruction`
+* `BlackTopHat`
+* `ClosingByReconstruction`
+* `DilateObjectMorphology`
+* `ErodeObjectMorphology`
+* `GrayscaleDilate`
+* `GrayscaleErode`
+* `GrayscaleMorphologicalClosing`
+* `GrayscaleMorphologicalOpening`
+* `MorphologicalGradient`
+* `OpeningByReconstruction`
+* `WhiteTopHat`
+
+
+FastMarchingImageFilter
++++++++++++++++++++++++
+
+The `FastMarchingImageFilter::AddTrailPoints` and `FastMarchingImageFilter::ClearTrialPoints` are removed. The `SetTrailPoints` method can be used instead to set the whole list of trial points.
```

Comment:
Trail -> Trial in `AddTrail` and `SetTrail`

Reply:
I am not sure what you are referring to here?

Reply:
The order of 'i' and 'a' is flipped in the `AddTrialPoints` and `SetTrialPoints`